libs/eavmlib/src/uart.erl

```diff
@@ -19,21 +19,38 @@
 %
 
 -module(uart).
```

Comment:
Looks fine, however, just to be sure, did you run the erlang formatter here?

Reply:
Ran erlfmt on part module and found one correction.